src/precord.c

```diff
@@ -44,6 +44,7 @@
 #ifdef HPCGAP
 #include "hpc/aobjects.h"
 #include "hpc/traverse.h"
+#include "hpc/guards.h"
```

Comment:
need to be moved up one line (clang-format sorts headers alphabetically)

Reply:
I have to do that for the headers alone, if I reformat all of `src/precord.c`, then it adds a lot of spurious changes (as the file does not currently match the clang-format style).